README.adoc

```diff
@@ -1,311 +1,313 @@
-
-// include::{chapters}extensions/shader_features.adoc[]
-
-== xref:{chapters}extensions/translation_layer_extensions.adoc[Translation Layer Extensions]
-
-  * `VK_EXT_custom_border_color`, `VK_EXT_border_color_swizzle`, `VK_EXT_depth_clip_enable`, `VK_EXT_depth_clip_control`, `VK_EXT_provoking_vertex`, `VK_EXT_transform_feedback`, `VK_EXT_image_view_min_lod`
-
-// include::{chapters}extensions/translation_layer_extensions.adoc[]
-
-== xref:{chapters}extensions/VK_EXT_descriptor_indexing.adoc[VK_EXT_descriptor_indexing]
-
-// include::{chapters}extensions/VK_EXT_descriptor_indexing.adoc[]
-
-== xref:{chapters}extensions/VK_EXT_inline_uniform_block.adoc[VK_EXT_inline_uniform_block]
-
-// include::{chapters}extensions/VK_EXT_inline_uniform_block.adoc[]
-
-== xref:{chapters}extensions/VK_EXT_memory_priority.adoc[VK_EXT_memory_priority]
-
-// include::{chapters}extensions/VK_EXT_memory_priority.adoc[]
-
-== xref:{chapters}extensions/VK_KHR_descriptor_update_template.adoc[VK_KHR_descriptor_update_template]
-
-// include::{chapters}extensions/VK_KHR_descriptor_update_template.adoc[]
-
-== xref:{chapters}extensions/VK_KHR_draw_indirect_count.adoc[VK_KHR_draw_indirect_count]
-
-// include::{chapters}extensions/VK_KHR_draw_indirect_count.adoc[]
-
-== xref:{chapters}extensions/VK_KHR_image_format_list.adoc[VK_KHR_image_format_list]
-
-// include::{chapters}extensions/VK_KHR_image_format_list.adoc[]
-
-== xref:{chapters}extensions/VK_KHR_imageless_framebuffer.adoc[VK_KHR_imageless_framebuffer]
-
-// include::{chapters}extensions/VK_KHR_imageless_framebuffer.adoc[]
-
-== xref:{chapters}extensions/VK_KHR_sampler_ycbcr_conversion.adoc[VK_KHR_sampler_ycbcr_conversion]
-
-// include::{chapters}extensions/VK_KHR_sampler_ycbcr_conversion.adoc[]
-
-== link:https://www.khronos.org/blog/vulkan-timeline-semaphores[VK_KHR_timeline_semaphore]
-
-== link:https://www.khronos.org/blog/streamlining-render-passes[VK_KHR_dynamic_rendering]
-
-== xref:{chapters}extensions/VK_KHR_shader_subgroup_uniform_control_flow.adoc[VK_KHR_shader_subgroup_uniform_control_flow]
-
-// include::{chapters}extensions/VK_KHR_shader_subgroup_uniform_control_flow.adoc[]
-
-= link:CONTRIBUTING.adoc[Contributing]
-
-= link:LICENSE[License]
-
-= link:CODE_OF_CONDUCT.adoc[Code of conduct]
+// Copyright 2019-2022 The Khronos Group, Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+= Vulkan^®^ Guide
+:regtitle: pass:q,r[^®^]
+The Khronos{regtitle} Vulkan Working Group
+:data-uri:
+:icons: font
+:max-width: 100%
+:numbered:
+:source-highlighter: rouge
+:rouge-style: github
+
+image::images/vulkan_logo.png[Vulkan Logo]
+image::images/khronos_logo.png[Khronos logo]
+
+:lang-jp: lang/jp/
+:lang-kor: lang/kor/
+This document is also available in the following languages: +
+xref:{lang-jp}README-jp.adoc[Japanese]
+xref:{lang-kor}README-kor.adoc[Korean]

```

Comment:
this should be the only line that is different, not sure why git thinks you changed the entire file

did you edit/save this file with a different encoding?

Reply:
I didn't do any encoding, but I did reset the file and re-commit.
Thanks for the detailed review.